src/utils/executeELM.js

```diff
@@ -152,33 +152,35 @@ async function executeELM(collector, oResourceTypes) {
           if (!evalResults[PATIENT_SUMMARY_KEY]) {
             evalResults[PATIENT_SUMMARY_KEY] = {};
           }
-          Promise.allSettled([
-            executeELMForReport(patientBundle),
-            ...elmLibs,
-          ]).then(
-            (results) => {
-              evalResults[PATIENT_SUMMARY_KEY]["ReportSummary"] =
-                results[0].status !== "rejected" ? results[0].value : null;
-              const evaluatedSurveyResults = executeELMForInstruments(
-                results.slice(1),
-                patientBundle
-              );
-              evalResults[PATIENT_SUMMARY_KEY]["SurveySummary"] =
-                evaluatedSurveyResults;
-              //debug
-              console.log(
-                "final evaluated CQL results including surveys ",
-                evalResults
-              );
-              resolve(evalResults);
-            },
-            (e) => {
-              console.log(e);
-              reject(
-                "Error occurred executing report library logics. See console for detail"
-              );
-            }
-          );
+          Promise.allSettled([executeELMForReport(patientBundle), ...elmLibs])
+            .then(
+              (results) => {
+                evalResults[PATIENT_SUMMARY_KEY]["ReportSummary"] =
+                  results[0].status !== "rejected" ? results[0].value : null;
+                const evaluatedSurveyResults = executeELMForInstruments(
+                  results.slice(1),
+                  patientBundle
+                );
+                evalResults[PATIENT_SUMMARY_KEY]["SurveySummary"] =
+                  evaluatedSurveyResults;
+                //debug
+                console.log(
+                  "final evaluated CQL results including surveys ",
+                  evalResults
+                );
+                resolve(evalResults);
+              },
+              (e) => {
+                console.log(e);
+                reject(
+                  "Error occurred executing report library logics. See console for detail"
+                );
+              }
+            )
+            .catch((e) => {
+              console.log("Error processing instrument ELM: ", e);
+              reject("rror processing instrument ELM. See console for details.");
```

Comment:
@achen2401  little typo here

Reply:
thank you for catching that @mcjustin.  Should be fixed now.